SuperBuild/External_JsonCpp.cmake

```diff
@@ -39,6 +39,7 @@ if(NOT DEFINED ${proj}_DIR AND NOT Slicer_USE_SYSTEM_${proj})
     SOURCE_DIR ${EP_SOURCE_DIR}
     BINARY_DIR ${EP_BINARY_DIR}
     CMAKE_CACHE_ARGS
+      -DCMAKE_POLICY_VERSION_MINIMUM:STRING=3.16.3
```

Comment:
```suggestion
```

Following integration of this pull request, this will not be required anymore:
* https://github.com/Slicer/Slicer/pull/8492

Additionally, we should:
* contribute a patch to upstream to update the oldest supported version
* backport it to our fork